### DISCUSSION TITLE
About 'Direction' Pole Figure

### DISCUSSION THREAD

**Discussion by louis96108** [mtex-toolbox/mtex#943] [Ask Anything] [answered]
Hi everyone,
I want to plot the 'Direction' Pole figure by certain Euler angles. I find that 'AztecCrystal'  has this function shown in the picture, but it's based on the EBSD data. I just have several sets of Euler angles. Could MTEX draw the ‘Direction’ Pole Figure ?
(https://user-images.githubusercontent.com/69557894/114172394-316fef80-9968-11eb-957d-f368affffd93.png)

Sincerely,
Louis

**Comment by ralfHielscher** [comment, marked as answer]
Sure,
```matlab
ori = orientation.byEuler(10*degree,20*degree,30*degree,csTit)
d = Miller(1,1,-2,0,csTit,'uvw');
plotPDF(ori,d)
```
Ralf.

**Reply by louis96108** [reply]
Hi, Ralf
Thanks for your reply. MTEX is really powerful software.

Cheers,
Louis
